src/generators/css-filter.ts

```diff
@@ -5,11 +5,23 @@ import {parseArray, formatArray} from '../utils/text';
 import {compareURLPatterns, isURLInList} from '../utils/url';
 import {createTextStyle} from './text-style';
 import {FilterConfig, InversionFix} from '../definitions';
+import {compareChromeVersions, getChromeVersion, isChromiumBased} from '../utils/platform';
 
 export enum FilterMode {
     light = 0,
     dark = 1
 }
+/**
+* Bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=501582
+* Patch: https://chromium-review.googlesource.com/c/chromium/src/+/1979258
+*/
```

Comment:
And this line should start with empty space

Reply:
It already starts with an empty space?

Reply:
My fault, it should start with single space. Compare:
```
/**
 * Correct
 */
```
```
/**
* Wrong
*/
```